### DISCUSSION TITLE
"Rejected Announce activity" form relay

### DISCUSSION THREAD

**Discussion by ksampolski** [mastodon/mastodon#21637] [Q&A]
Hello guys! I started my own instance and have a problem. I get all the toots from the relay and everything works fine, but when I post something on my timeline I get feedback from the relay: 
`Rejected Announce activity https://relay.101010.pl/activities/898ef4bb-067c-4c83-9096-a102e3f57f08 from https://relay.101010.pl/actor`

This is strange because the relay is added as https://relay.101010.pl/inbox and not https://relay.101010.pl/actor. I wonder if the problem is on my side or on the side of the relay. I'm currently using mastodon version 4.0.2

**Comment by lifeofguenter** [comment]
Try another relay: https://github.com/brodi1/activitypub-relays
